src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import FONTFAMILY from "../../../variables/font_family";
+import plus from "../../../assets/svg/plus.svg";
+
+const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledBtn onClick={onClick}>
+      <HoverCircle></HoverCircle>
+      <Img src={plus} />
+      <Mess>タスクを追加</Mess>
+    </StyledBtn>
+  );
+};
+
+const StyledBtn = styled.button`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 126px;
+  height: 24px;
+  border-radius: 12px;
+  padding: 0;
+  border: none;
+  background-color: transparent;
+`;
+
+const HoverCircle = styled.div`
+  position: absolute;
+  width: 126px;
+  height: 24px;
```

Comment:
```suggestion
  width: 100%;
  height: 100%;
```
意図としては「126pxにしたい」というよりも「親要素と同じにしたい」なので、こちらがいいのかなと思います。
（ボタン自体の大きさが変わればそれにつれて影の大きさも変化するのが自然な気がします）